data/nlu/nlu.md

```diff
@@ -3045,6 +3046,7 @@
 - we think 4 million INR/ year
 - we're building a conversational assistant for our employees to book meeting rooms.
 - £50k
+- botonic
```

Comment:
is this in the context of a user giving their company? if so we should annotate it as such

Reply:
(same with wordpress and slack, actually)

Reply:
Both were in the context of a user providing information related to switching from another platform. The story expects the `enter_data` intent